installer/templates/phx_web/controllers/page_html/home.html.heex

```diff
@@ -116,33 +115,66 @@ export PATH=$HOME/<%= @elixir_install_bin_path %>:$PATH</code><% end %>
         fill="#FD4F00"
       />
     </svg>
-    <h1 class="text-brand mt-10 flex items-center text-sm font-semibold leading-6">
-      Phoenix Framework
-      <small class="bg-brand/5 text-[0.8125rem] ml-3 rounded-full px-2 font-medium leading-6">
-        v{Application.spec(:phoenix, :vsn)}
-      </small>
-    </h1>
-    <p class="text-[2rem] mt-4 font-semibold leading-10 tracking-tighter text-zinc-900 text-balance">
+    <div class="mt-10 flex justify-between items-center">
+      <h1 class="flex items-center text-sm font-semibold leading-6">
+        Phoenix Framework
+        <small class="bg-warning text-warning-content text-[0.8125rem] ml-3 rounded-box px-2 font-medium leading-6">
```

Comment:
Consider using the `badge` component

`class="badge badge-warning badge-sm ml-3`

https://daisyui.com/components/badge/